packages/server/api/src/app/ee/platform/platform-plan/platform-plan-helper.ts

```diff
@@ -61,4 +69,178 @@ export async function checkQuotaOrThrow(params: QuotaCheckParams): Promise<void>
             },
         })
     }
+}
+
+export async function checkProjectNotLockedOrThrow(projectId?: string): Promise<void> {
```

Comment:
Do not export function like this, you should wrap it around const?

for exmaple

export const platformUsageHelper = {

}